### PR TITLE
Pretty language i18n dropdown

### PR DESCRIPTION
**Pull request by TheRolfFR**
To resole #133 , I made a smoll dropdown:

image

I chose this location because it does not take space on the name or login button

It doesn't have country full names but it does have flags from https://flagpedia.net/download/api

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -131,6 +131,39 @@
 								</v-btn>
 							</v-card>
 
+                     <v-menu offset-y>
+                        <template v-slot:activator="{ on, attrs }">
+                           <v-btn
+                              class="ml-2"
+                              v-bind="attrs"
+                              v-on="on"
```
Comment (3vorp) [inline index.html:139]: should have same border radius as all the other buttons

File: index.html [hunk 2]
```diff
@@ -131,6 +131,39 @@
...
+                              v-for="language in languages"
+                              :key="language.bcp47"
+                              :class="{ 'primary': language.lang == selectedLang }"
+                              v-on:click="() => selectedLang = language.lang"
+                              style="cursor: pointer"
+                              dense
+                           >
+                              <v-list-item-avatar min-width="24" :width="24" max-width="24" :height="18" class="rounded-0">
+                                 <v-img
+                                    :src="'https://flagcdn.com/24x18/' + language.iso3166 + '.png'"
```
Comment (3vorp) [inline index.html:156]: use string interpolation (e.g. `https://flagcdn.com/24x18/${language.iso3166}.png`), much prettier

**Review by 3vorp — CHANGES_REQUESTED** [review]
The menu itself looks good, but I feel like the button is really randomly placed and feels very inconsistent. The lack of a background in dark mode also makes it somewhat hard to read, plus it looks like it's part of the theme selector since you put them so close together.

I was thinking we could either put the language selector on the toolbar (next to the faithful site redirect) or on the bottom corner of the sidebar (where it used to be but with a button)

File: index.html (hunk 1 above)
Comment (TheRolfFR) [reply index.html:139]: Button was transparent so whatever and will be placed somewhere else

**Comment by TheRolfFR** [comment]
> The menu itself looks good, but I feel like the button is really randomly placed and feels very inconsistent. 

Huh you made a screenshot placing them that close to each other in the [issue](#133 ).

> The lack of a background in dark mode also makes it somewhat hard to read, plus it looks like it's part of the theme selector since you put them so close together.

Agree but these are the only two settings we have: theme + language

> I was thinking we could either put the language selector on the toolbar (next to the faithful site redirect) or on the bottom corner of the sidebar (where it used to be but with a button)

The toolbar may be more appropriate but we should also relocate and redesign the theme button at the same time.
A good example is the v2 vuetify website:
image image
Our theme selector will be the dropdown just like here

**Comment by TheRolfFR** [comment]
New menus look like this in app bar:
image image

**Comment by 3vorp** [comment]
I feel like the faithful website button should go at the end but I really like this 👍

**Comment by TheRolfFR** [comment]
Agree, should we give it its color back and a colored button to emphasize it ? he seems lonely and sad in white

What about translation labels ? Do we keep the two-letter symbols or should we write the full language name?

**Comment by 3vorp** [comment]
I think it's fine in monochrome; it fits better with the other icons at any rate. The two letter symbols seem clear enough to me, but to be consistent with the theme selector it might be better to use the full name.

**Review by 3vorp — APPROVED** [review]

**Pull request merged**